### PR TITLE
Add TensorRT examples 

### PR DESCRIPTION
**Pull request by xiaolul**
- put all necessary model related code to `model_common`
- put a readme file in the pytensorrt-text-detection folder
- cleanup the jupyter notebook

*Intentionally keep the ipynb format*

### CODE REVIEW

File: py-tensorrt-text-detection/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+## Development Environment
+### Install TensorRT
+#### Method 1: Native Install
+Based on the [installation guide](https://docs.nvidia.com/deeplearning/tensorrt/install-guide/index.html#installing-pip), we can install tensorRT using pip
+alone. 
+1. (If you are working on conda virtual environment), follow steps in 
+    [this page](https://github.com/stan-dev/pystan/issues/294).
+1. Install pycuda using pip, version < 2021.1 
+1. Pip install tensorR (change the tensorR version to the expected one, suggest no earlier than 7.0 version)
```
Comment (tsuting) [inline py-tensorrt-text-detection/README.md]: tensorRT? the same for the next one

File: py-tensorrt-text-detection/README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    ```bash
+    python3 -m pip install --upgrade setuptools pip
+    python3 -m pip install nvidia-pyindex
+    python3 -m pip install --upgrade nvidia-tensorrt
+    ```
+    If you are using a requirements.txt to install tensorR package, plase make sure `--extra-index-url https://pypi.ngc.nvidia.com` is added to your requirements.txt file.
```
Comment (tsuting) [inline py-tensorrt-text-detection/README.md]: and here lol

File: py-tensorrt-text-detection/README.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+## Tested OS Environment
+- We tested the code on
+    - Desktop or VM using Ubuntu 20.10 (with tools installed of version Ubuntu 20.04 version since 20.10 is not officially supported) and Ubuntu 18.04 OS.
+    - Jetson Nano device.
+
+- We *did not* test on
+    - Windows OS
+    - MacOS
+
+- Using WSL (Windows Subsystem Linux) is NOT RECOMMENDED. You may run into unknown error messages.
```
Comment (tsuting) [inline py-tensorrt-text-detection/README.md]: The error stated here https://docs.nvidia.com/deeplearning/tensorrt/release-notes/tensorrt-8.html#rel-8-4-0-EA
in the section "Known Issues". Could you help me add a link?

File: py-tensorrt-text-detection/README.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+│   ├── README.md
+│   ├── requirements.txt
+│   ├── tensorrt_notebook.ipynb
+│   └── trt_utils.py
+├── requirements.txt
+└── sample_images
+    └── nutrition_label.jpg
+```
+- `trt_utils.py` is the python script showing how to convert an onnx model to tensorRT engine using python API and then run single inference using the converted tensorRT engine. Note that, the onnx to tensorRT engine conversion is tailored to paddlepaddle text detection model,  PLEASE MODIFY IT ACCORDINGLY BASED ON YOUR MODEL.
+- `tensorrt_notebook.ipynb` is a python notebook that demostrates the model conversion and inference process using tensorRT engine and PaddlePaddle text detection model.
```
Comment (tsuting) [inline py-tensorrt-text-detection/README.md]: demonstrate

**Review by tsuting — APPROVED** [review]
Looks great. Thanks.

File: py-tensorrt-text-detection/README.md (hunk 1 above)
Comment (xiaolul) [reply py-tensorrt-text-detection/README.md]: ..not sure why "T" is missing, added.

File: py-tensorrt-text-detection/README.md (hunk 4 above)
Comment (xiaolul) [reply py-tensorrt-text-detection/README.md]: a notebook demonstrates 

isn't it correct?

**Pull request merged**